pages/implementation/reference/javascript.mdx

```diff
@@ -0,0 +1,196 @@
+---
+layout: 'guide'
+title: 'JavaScript'
+---
+
+import Link from '../../../components/Link';
+
+### Platforms
+
+Avo can generate JavaScript code targeted at the following platforms
+
+- Web
+- React Native
+- Node.js
+
+### Quickstart
```

Comment:
Maybe split quickstart and reference to separate docs 🤔

Reply:
Good question! Not sure where the quickstart vs reference would live. I don't think we'll be mapping out all the languages in other places in the doc? Maybe stick with it as is for now?